core/sql/executor/ExExeUtilGet.cpp

```diff
@@ -266,6 +266,12 @@ static const QueryString getComponentPrivileges[] =
   {" ; "}
 };
 
+static const QueryString getCatalogsQuery[] =
+{
+  {" select * from (values ('TRAFODION'), ('HIVE')) "},
```

Comment:
Currently, trafodion and hive are true catalogs that can be traversed through DBMgr
or GET statements. Details about their contents (schemas, tables, etc) can
be returned.
'hbase' is not really a catalog, it is the first part of a 3-part name used
to access native hbase tables. We can think about it if it makes sense to treat it
as a catalog.